README.md

```diff
@@ -15,61 +21,307 @@ Our other repositories are:
+configuration after installation to get working out of the box:
 
-  `make run-flask`
+- [jq](https://stedolan.github.io/jq/) - for working with JSON in the command
+  line
+- [git](https://git-scm.com/) - for version control management
+- [tfenv](https://github.com/tfutils/tfenv) - for managing
+  [Terraform](https://www.terraform.io/) installations
+- [cf-cli@8](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) - for
+  working with a Cloud Foundry platform (e.g., cloud.gov)
+- [redis](https://redis.io/) - required as the backend for the API's
+  asynchronous job processing
+- [vim](https://www.vim.org/) - for editing files more easily in the command
+  line
 
-- Run Celery (background worker)
+You can install them by running the following:
 
-  `make run-celery`
+```sh
+brew install jq git tfenv cloudfoundry/tap/cf-cli@8 redis vim
+```
+
+#### Terraform Installation
+
+As a part of the installation above, you just installed `tfenv` to manage
+Terraform installations. This is great, but you still need to install Terraform
+itself, which can be done with this command:
+
+```sh
+tfenv install latest:^1.4.0
+```
+
+_NOTE: This project currently uses the latest `1.4.x release of Terraform._
 
-### Python dependency management
+#### Python Installation
+
+Now we're going to install a tool to help us manage Python versions on our
+system.  First, we'll install [pyenv](https://github.com/pyenv/pyenv) with
+Homebrew:
+
+```sh
+brew install pyenv
+```
+
+When that finishes installing, you'll need to make another adjustment in the
+file that you adjusted for your `$PATH` environment variable and Homebrew's
+setup. Open the file, and add these lines to it:
+
+```
+# pyenv setup
+export PYENV_ROOT="$HOME/.pyenv"
+command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init -)"
+```
+
+Once again, start a new shell session or source the file in your current shell
+session to make the changes take effect.
+
+Now we're ready to install the Python version we need with `pyenv`, like so:
+
+```sh
+pyenv install 3.9
+```
+
+_NOTE: This project currently runs on Python 3.9.x._
+
+This will install the latest version of Python 3.9.
+
+#### Python Dependency Installation
+
+Lastly, we need to install the tool we use to manage Python dependencies within
+the project, which is [poetry](https://python-poetry.org/).
+
+Visit the
+[official installer instructions page](https://python-poetry.org/docs/#installing-with-the-official-installer)
+and follow the steps to install Poetry directly with the script.
+
+This will ensure `poetry` doesn't conflict with any project virtual environments
+and can update itself properly.
+
+#### PostgreSQL installation
+
+We now need to install a database - this project uses PostgreSQL, and Homebrew
+requires a version number to be included with it when installing it:
+
+_NOTE: This project currently works with PostgreSQL version 15.x_
```

Comment:
We're mostly devving on 15, but the servers are currently running 12

Reply:
Ah thanks for calling that out!  We should see if we can get some updates going on those - it's now possible to do ourselves. 🙂

Reply:
Updated this to note version 12.x for hosted environments currently.